addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13850,7 +13851,12 @@ msgctxt "#24147"
 msgid "Failed to scan %s: %s"
 msgstr ""
 
-#empty strings from id 24148 to 24991
+#empty strings from id 24148 to 24990
+
+#: xbmc/filesystem/AddonsDirectory.cpp
```

Comment:
would you mind adding a little contextual note, like if this is a error message, dialogue headline and maybe in which cases it's triggered (if relevant for translation)? Translators won't dig the source to understand how stuff is used, so for them a little contextual help is valuable for correct translations. Something in the direction:
`#. Error message that's shown when ...`